libs/data-types/src/lib/types/CoscradDataType.ts

```diff
@@ -1,5 +1,8 @@
 enum CoscradDataType {
     NonEmptyString = 'NON_EMPTY_STRING',
+    UUID = 'UUID',
+    URL = 'URL',
+    NonNegativeFiniteNumber = 'NonNegativeFiniteNumber',
```

Comment:
Why is this camelcase rather than caps-kabob?

Reply:
Ooops. Fixed.